rx/concurrency/mainloopscheduler/wxscheduler.py

```diff
@@ -56,10 +56,8 @@ def interval() -> None:
                 sad.disposable = action(scheduler, state)
 
         msecs = int(self.to_seconds(time) * 1000.0)
-        if msecs == 0:
-            msecs = 1  # wx.Timer doesn't support zero.
```

Comment:
I think it might have applied for older versions — but the docs don’t mention this caveat and it seemed to work fine when I tried it.

https://wxpython.org/Phoenix/docs/html/wx.Timer.html#wx.Timer.Start

Or should we play it safe in case people have older wx?

Reply:
Actually version 3 also doesn't mention zero being an invalid value:
https://docs.wxwidgets.org/3.0/classwx_timer.html#a8df981f7075786811598828af1d294ac
@bosonogi you wrote this originally -- do you think it's all right to remove the guard against zero delay?

Reply:
Ok thanks for your insight :+1:  I guess that should be ok.

Reply:
> Actually version 3 also doesn't mention zero being an invalid value:
> https://docs.wxwidgets.org/3.0/classwx_timer.html#a8df981f7075786811598828af1d294ac
> @bosonogi you wrote this originally -- do you think it's all right to remove the guard against zero delay?

@erikkemperman wx has different implementations of `wxTimer` on different platforms it supports. Perhaps only some of them had issues with zero. I was working on Windows and OSX at the time, and I remember having to include this guard for the scheduler to work properly.

I'm all for removing it if it's no longer necessary, but it should be tested on more then one platform.

Reply:
@bosonogi Thanks for the quick response! It was tested so far on Linux and MacOS, unfortunately I don't have a Windows box... If anyone does and could check if this works that would be great!

Reply:
For what it’s worth: I managed to get wxpython 4.0.4 working on Travis’ Windows (and the other platforms) and the “zero tests” work fine.